runtime/caml/backtrace_prim.h

```diff
@@ -110,8 +111,35 @@ value caml_remove_debug_info(code_t start);
  * We use `intnat` for max_frames because, were it only `int`, passing
  * `max_int` from the OCaml side would overflow on 64bits machines. */
 
-intnat caml_current_callstack_size(intnat max_frames);
-void caml_current_callstack_write(value trace);
+enum { Small_callstack_size = 32 };
```

Comment:
Habit mostly? There are minor advantages for using an enum in C to define integer constants: it's a real symbol (so it's available in gdb, unlike `#define`), and it doesn't have linkage (so it works in header files, unlike `const uintnat`).

I think `#define` is more idiomatic for this codebase though, I'll change it.

Reply:
Another reason in favour of `enum`: unlike `const uintnat` it defines a genuine constant, so it can be used to give the dimension of the `init` array in the `elems` union below.